backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/CassandraTableManager.java

```diff
@@ -60,27 +59,26 @@ public CassandraTableManager ensureAllTables() {
 
     public void clearAllTables() {
         CassandraAsyncExecutor executor = new CassandraAsyncExecutor(session);
-        FluentFutureStream.of(
-            module.moduleTables()
-                .stream()
+        Flux.fromIterable(module.moduleTables())
                 .map(CassandraTable::getName)
-                .map(name -> truncate(executor, name)))
-            .join();
+                .flatMap(name -> truncate(executor, name))
+                .publishOn(Schedulers.elastic())
```

Comment:
you probably have to declare the `publishOn` before the flatmap.
Did you check with log ?

Reply:
Nothing in logs.

Reply:
what does it mean ? the logs should tell you where things are run

Reply:
Either the logs may not have been activated, or I do not know what to look for.

Reply:
I explained that yesterday:
* put `.log` at the right locations
* configure logback to output the logger with name="reactor" in DEBUG